projects/storefrontapp-e2e-cypress/cypress/helpers/product-configurator-vc.ts

```diff
@@ -210,11 +210,11 @@ export function checkConflictDescriptionDisplayed(description: string): void {
 }
 
 /**
- * Navigates to the corresponding group that contains an attribute which is involved in a conflict.
+ * Navigates to the configuration group that contains an attribute which is involved in a conflict.
```

Comment:
Why change product configuration tests?  This doesn't seem to fit the scope of the PR.
Let's discuss.

Reply:
Don't know why this is here. I didn't make any changes to this file. Maybe during a /develop branch merge it changed these files